lib/rubocop_director/commands/generate_config.rb

```diff
@@ -8,6 +8,11 @@ class GenerateConfig
 
       RUBOCOP_TODO = ".rubocop_todo.yml"
 
+      def initialize(director_config:, todo_config: TODO_CONFIG_NAME)
```

Comment:
Since we agreed to make todo path not configurable, I guess we can remove it from the initializer and just use the constant

Reply:
yes agreed, I somehow missed this while writing test cases. thanks!